src/length.js

```diff
@@ -1,5 +1,5 @@
 var _curry1 = require('./internal/_curry1');
-var is = require('./is');
+var _isNumber = require('./internal/_isNumber');
```

Comment:
How did we get ourselves to the state where we proudly promote a public API of `R.is(Type, val)` and internally have a dozen `internal/isTypeA)` calls.  It just feels strange.